public/assets/css/main.css

```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
```

Comment:
Why not put this in footer ul ?

Reply:
The same footer is used for multiple pull requests. 😓 In #19 we have multiple lists and some aren't in the footer.